dev-lua/lua-openssl/lua-openssl-0.4.1.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit toolchain-funcs
+
+DESCRIPTION="OpenSSL binding for Lua"
+HOMEPAGE="https://github.com/zhaozg/lua-openssl"
+SRC_URI="https://github.com/zhaozg/lua-openssl/archive/${PV}.tar.gz -> lua-openssl-${PV}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="luajit libressl"
+
+RDEPEND="
+	luajit? ( dev-lang/luajit:2 )
+	!luajit? ( >=dev-lang/lua-5.1 )"
+DEPEND="${RDEPEND}
+	libressl? ( dev-libs/libressl )
+	!libressl? ( dev-libs/openssl:0 )
+	virtual/pkgconfig"
+
+use luajit && LUAV=luajit || LUAV=lua
```

Comment:
That is absolutely not allowed. No use-conditionals in global scope. You can do that in pkg_setup or src_prepare